core/src/test/resources/org/apache/calcite/test/TopDownOptTest.xml

```diff
@@ -441,223 +441,223 @@ EnumerableMergeJoin(condition=[AND(=($1, $4), =($0, $3))], joinType=[inner])
   EnumerableSort(sort0=[$1], sort1=[$0], dir0=[ASC], dir1=[ASC])
     EnumerableTableScan(table=[[CATALOG, SALES, BONUS]])
 ]]>
-    </Resource>
-  </TestCase>
-  <TestCase name="testSortAggPartialKey">
-    <Resource name="sql">
-      <![CDATA[select mgr,deptno,comm,count(*) from sales.emp
+        </Resource>
+    </TestCase>
+    <TestCase name="testSortAggPartialKey">
+        <Resource name="sql">
+            <![CDATA[select mgr,deptno,comm,count(*) from sales.emp
 group by mgr,deptno,comm
 order by comm desc nulls last, deptno nulls first]]>
-    </Resource>
-    <Resource name="planBefore">
-      <![CDATA[
+        </Resource>
+        <Resource name="planBefore">
+            <![CDATA[
 LogicalSort(sort0=[$2], sort1=[$1], dir0=[DESC-nulls-last], dir1=[ASC-nulls-first])
   LogicalAggregate(group=[{0, 1, 2}], EXPR$3=[COUNT()])
     LogicalProject(MGR=[$3], DEPTNO=[$7], COMM=[$6])
       LogicalTableScan(table=[[CATALOG, SALES, EMP]])
 ]]>
-    </Resource>
-    <Resource name="planAfter">
-      <![CDATA[
+        </Resource>
+        <Resource name="planAfter">
+            <![CDATA[
```

Comment:
hmm.. Looks like we have different style settings.
Can you remove these diffs?

Reply:
+1. Introduce lost of changes just due to indention seems not good.

Reply:
BTW, there is another JIRA CALCITE-4027, do you mind taking it, @amaliujia ?

Reply:
Sure. Will take a look.

Reply:
thanks!

Reply:
Oops, will do.